dist/PublicLab.Editor.js

```diff
@@ -22804,6 +22807,21 @@ module.exports = function CustomInsert(_module, wysiwyg) {
       }
     });
   });
+
+  $(':not(".woofmark-command-insert")').click((e) => {
+    // check to see that the clicked element isn't fa-insert icon, else when you click on the fa-insert icon, the popover will close
+    try {
+      if ($('.woofmark-command-insert').children()[0] != e.target) {
```

Comment:
I think you should also check if the popover is open here by adding 

```diff
+  if (popoverIsOpen && $('.woofmark-command-insert').children()[0] != e.target) {
```

this way, you don't need a try/catch block @RaviAnand111

Reply:
Oh, Ok I will be adding that in the code, Thanks